lib/src/ast.dart

```diff
@@ -434,3 +434,11 @@ class _ElementVisitorAdapter extends GeneralizingElementVisitor {
     }
   }
 }
+
+extension ElementExtension on Element? {
+  bool get isDartCorePrint {
+    var element = this;
```

Comment:
In analyzer we prefer making this `final self = this;`.

Reply:
Ah, nice convention.